<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorState</class>
 <widget class="QWidget" name="EditorState">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0" colspan="7">
      <widget class="QLineEdit" name="nameField"/>
     </item>
     <item row="5" column="5" colspan="2">
      <widget class="QListView" name="outActions"/>
     </item>
     <item row="4" column="6">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="moveOutUp">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ui.qrc">
           <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="moveOutDown">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ui.qrc">
           <normaloff>:/images/down.png</normaloff>:/images/down.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&amp;Available</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>availableActions</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="7">
      <widget class="QPlainTextEdit" name="commentField"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;In</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>inActions</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="4">
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="addOut">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ui.qrc">
           <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeOut">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ui.qrc">
           <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="moveInUp">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ui.qrc">
           <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="moveInDown">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ui.qrc">
           <normaloff>:/images/down.png</normaloff>:/images/down.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&amp;Out</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>outActions</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="addIn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ui.qrc">
           <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeIn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ui.qrc">
           <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="3">
      <widget class="QListView" name="availableActions"/>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QListView" name="inActions"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>moveInUp</sender>
   <signal>clicked()</signal>
   <receiver>EditorState</receiver>
   <slot>moveInUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>779</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveInDown</sender>
   <signal>clicked()</signal>
   <receiver>EditorState</receiver>
   <slot>moveInDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>775</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveOutUp</sender>
   <signal>clicked()</signal>
   <receiver>EditorState</receiver>
   <slot>moveOutUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>848</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>610</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveOutDown</sender>
   <signal>clicked()</signal>
   <receiver>EditorState</receiver>
   <slot>moveOutDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addIn</sender>
   <signal>clicked()</signal>
   <receiver>EditorState</receiver>
   <slot>addIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeIn</sender>
   <signal>clicked()</signal>
   <receiver>EditorState</receiver>
   <slot>removeIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addOut</sender>
   <signal>clicked()</signal>
   <receiver>EditorState</receiver>
   <slot>addOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeOut</sender>
   <signal>clicked()</signal>
   <receiver>EditorState</receiver>
   <slot>removeOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>moveInUp()</slot>
  <slot>moveInDown()</slot>
  <slot>moveOutUp()</slot>
  <slot>moveOutDown()</slot>
  <slot>addIn()</slot>
  <slot>addOut()</slot>
  <slot>removeIn()</slot>
  <slot>removeOut()</slot>
 </slots>
</ui>
